src/core/Expander/Expander/Expander.md

```diff
@@ -80,64 +243,235 @@ import {
+          e-identification.
+        </Paragraph>
+      </ExpanderContent>
     </Expander>
-    <Expander>
-      <ExpanderTitleButton>Test expander 2</ExpanderTitleButton>
-      <ExpanderContent>Test expander content 2</ExpanderContent>
+    <Expander open={expanderTwoOpen}>
+      <ExpanderTitleButton>
+        Is the application secure?
+      </ExpanderTitleButton>
+      <ExpanderContent>
+        <Paragraph marginBottomSpacing="l">
+          Suomi.fi Messages is a secure way to communicate with
+          organisations that use the service.
+          <ul>
+            <li>
+              The messages and attachments shown in the application
+              are transmitted in encrypted form between the mobile
+              device and the server.
+            </li>
+            <li>
+              The app requires permissions to use your mobile device’s
+              photographs and other files, so it can process
+              attachments included in messages.
+            </li>
+            <li>
+              The user is individually identified using strong
+              identification. This means that you are the only person
+              who can access your data.
+            </li>
+          </ul>
+        </Paragraph>
+        <Paragraph>
+          If you break your mobile device or lose it, you can sign out
+          of the Suomi.fi application either in the Suomi.fi web
+          service or by using another phone.
+        </Paragraph>
+      </ExpanderContent>
     </Expander>
-    <Expander
-      open={expanderThreeOpen}
-      onOpenChange={(open) => {
-        if (window.confirm('Toggle Expander 3')) {
-          setExpanderThreeOpen(!open);
-        }
-      }}
-    >
-      <ExpanderTitleButton>Test expander 3</ExpanderTitleButton>
-      <ExpanderContent>Test expander content 3</ExpanderContent>
+    <Expander open={expanderThreeOpen}>
+      <ExpanderTitleButton>
+        Errors in the Suomi.fi mobile application
+      </ExpanderTitleButton>
+      <ExpanderContent>
+        <Paragraph marginBottomSpacing="l">
+          If the application does not work on your mobile device for
+          some reason, you can also read your messages via a browser
+        </Paragraph>
+        <Paragraph>
+          Errors occurring in the Suomi.fi application generally do
+          not affect the preservation of Suomi.fi Messages.
+        </Paragraph>
+      </ExpanderContent>
     </Expander>
   </ExpanderGroup>
 </>;
 ```
+
+### Interactive elements in Expander title
+
+If you need to have interactive elements in the Expander title, use `<ExpanderTitle>` instead of `<ExpanderTitleButton>`.
+
+ExpanderTitle creates a smaller toggle button to the right side of the Expander. It is important to set `ariaOpenText`/`ariaCloseText` props as well as connect the interactive element to the button with `toggleButtonAriaDescribedBy` to provide adequate context for assistive technologies.
```

Comment:
Because aria-expanded already communicates state, it can be confusing to change the text too. This has been reported separately as a bug of the component itself, but may I suggest a temporary "fix" of simply making the example text the same?

 ```suggestion
ExpanderTitle creates a smaller toggle button to the right side of the Expander. Generally `ariaOpenText`/`ariaCloseText` props should have the same value, because state is already communicated to assistive technology using `aria-expanded`. Point `toggleButtonAriaDescribedBy` to the ID of interactive element to provide additional context for assistive technology.
```

Reply:
I say let's go with this docs change for now. 

Changed it like you suggested.